### DISCUSSION TITLE
Does .net7 officially intend to support NativeAOT-LLVM?

### DISCUSSION THREAD

**Discussion by GreenShadeZhang** [dotnet/runtime#71416] [Q&A] [answered]
I saw NativeAOT-LLVM in the runtimelab repository, but .net7 is about to be released, so I wonder if there is any hope of adding this feature?

**Comment by hez2010** [comment, marked as answer]
NativeAOT-LLVM is an experimental project specialized for WebAssembly target. I don't think it will be supported by .NET 7.

**Reply by GreenShadeZhang** [reply]
Okay, thanks for answering.

**Reply by GreenShadeZhang** [reply]
If I want to compile C# code directly into wasm, is there any complete way?
